homeassistant/components/onkyo/config_flow.py

```diff
@@ -165,6 +167,49 @@ async def async_step_eiscp_discovery(
             ),
         )
 
+    async def async_step_ssdp(
+        self, discovery_info: ssdp.SsdpServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle flow initialized by SSDP discovery."""
+        _LOGGER.debug("Config flow start ssdp: %s", discovery_info)
+
+        if udn := discovery_info.ssdp_udn:
+            udn_parts = udn.split(":")
+            if len(udn_parts) == 2:
+                uuid = udn_parts[1]
+                last_uuid_section = uuid.split("-")[-1].upper()
+                await self.async_set_unique_id(last_uuid_section)
+                self._abort_if_unique_id_configured()
+
+        if discovery_info.ssdp_location is None:
+            _LOGGER.error("SSDP location is None")
+            return self.async_abort(reason="unknown")
+
+        host = URL(discovery_info.ssdp_location).host
+
+        if host is None:
+            _LOGGER.error("SSDP host is None")
+            return self.async_abort(reason="unknown")
+
+        try:
+            info = await async_interview(host)
+        except OSError:
+            _LOGGER.exception("Unexpected exception interviewing host %s", host)
+            return self.async_abort(reason="unknown")
+
+        if info is None:
+            _LOGGER.debug("SSDP eiscp is None: %s", host)
+            return self.async_abort(reason="cannot_connect")
+
+        await self.async_set_unique_id(info.identifier)
+        self._abort_if_unique_id_configured()
+
+        self._receiver_info = info
+
+        title_string = f"{info.model_name} ({info.host})"
+        self.context.update({"title_placeholders": {"name": title_string}})
```

Comment:
```suggestion
        self.context["title_placeholders"] = {"name": title_string}
```

Since the whole value is replaced, `update` isn't needed and its faster to set it instead